pkg/controller/cassandracluster/testdata/cassandracluster-2DC-configmap.yaml

```diff
@@ -0,0 +1 @@
+../../../../test/e2e/testdata/cassandracluster-2DC-configmap.yaml
```

Comment:
Are you sure you can do something like that ?

Reply:
it's a symlink